Plugins/GitUIPluginInterfaces/ILazyGitRefsProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace GitUIPluginInterfaces
+{
+    /// <summary>
+    /// A lazy provider for GitRefs() that can be shared for instance when a repository is changed.
+    /// </summary>
+    public interface ILazyGitRefsProvider
```

Comment:
❔
```suggestion
    public interface IFilteredGitRefsProvider
```
This interface stands for itself. It is kind of lazy implicitly. The degree of laziness depends on the implementation.

Reply:
ok